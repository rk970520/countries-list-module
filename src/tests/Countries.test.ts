import { Countries } from '../index';
var jsonData:Array<object> = require('../countries.json');
test('Countries Test', () => {
  Countries(jsonData).forEach((country:object) => {
        expect(country).toMatchSnapshot({
          name: expect.any(String),
          code: expect.any(String)
        });
      });
});