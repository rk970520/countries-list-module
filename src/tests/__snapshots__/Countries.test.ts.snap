// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Countries Test 1`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 2`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 3`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 4`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 5`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 6`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 7`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 8`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 9`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 10`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 11`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 12`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 13`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 14`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 15`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 16`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 17`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 18`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 19`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 20`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 21`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 22`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 23`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 24`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 25`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 26`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 27`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 28`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 29`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 30`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 31`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 32`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 33`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 34`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 35`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 36`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 37`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 38`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 39`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 40`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 41`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 42`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 43`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 44`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 45`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 46`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 47`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 48`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 49`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 50`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 51`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 52`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 53`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 54`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 55`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 56`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 57`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 58`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 59`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 60`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 61`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 62`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 63`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 64`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 65`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 66`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 67`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 68`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 69`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 70`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 71`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 72`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 73`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 74`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 75`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 76`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 77`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 78`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 79`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 80`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 81`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 82`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 83`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 84`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 85`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 86`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 87`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 88`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 89`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 90`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 91`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 92`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 93`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 94`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 95`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 96`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 97`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 98`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 99`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 100`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 101`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 102`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 103`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 104`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 105`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 106`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 107`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 108`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 109`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 110`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 111`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 112`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 113`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 114`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 115`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 116`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 117`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 118`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 119`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 120`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 121`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 122`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 123`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 124`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 125`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 126`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 127`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 128`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 129`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 130`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 131`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 132`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 133`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 134`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 135`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 136`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 137`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 138`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 139`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 140`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 141`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 142`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 143`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 144`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 145`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 146`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 147`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 148`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 149`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 150`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 151`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 152`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 153`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 154`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 155`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 156`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 157`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 158`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 159`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 160`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 161`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 162`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 163`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 164`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 165`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 166`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 167`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 168`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 169`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 170`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 171`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 172`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 173`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 174`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 175`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 176`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 177`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 178`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 179`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 180`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 181`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 182`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 183`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 184`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 185`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 186`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 187`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 188`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 189`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 190`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 191`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 192`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 193`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 194`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 195`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 196`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 197`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 198`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 199`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 200`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 201`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 202`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 203`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 204`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 205`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 206`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 207`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 208`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 209`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 210`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 211`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 212`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 213`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 214`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 215`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 216`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 217`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 218`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 219`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 220`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 221`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 222`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 223`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 224`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 225`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 226`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 227`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 228`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 229`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 230`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 231`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 232`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 233`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 234`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 235`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 236`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 237`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 238`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 239`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 240`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 241`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 242`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;

exports[`Countries Test 243`] = `
Object {
  "code": Any<String>,
  "name": Any<String>,
}
`;
